packages/sns/src/sns.wrapper.ts

```diff
@@ -126,6 +141,100 @@ export class SnsWrapper {
     params: Omit<SnsGetNeuronParams, "certified">
   ): Promise<Neuron> => this.governance.getNeuron(this.mergeParams(params));
 
+  /**
+   * Returns the subaccount of the next neuron to be created.
+   *
+   * The neuron account is a subaccount of the governance canister.
+   * The subaccount is derived from the controller and an ascending index.
+   *
+   * @param controller
+   * @returns
+   */
+  getNextNeuronAccount = async (
+    controller: Principal
+  ): Promise<{ account: SnsAccount; index: bigint }> => {
+    for (let index = 0; index < MAX_NEURONS_SUBACCOUNTS; index++) {
```

Comment:
we must do this in a procedural way? wouldn't it be more performant to `Promise.all` the calls?

Reply:
We need the index and also we don't know which index will be the one with no balance. Therefore, I used a for loop.

I guess we could always improve the performance by parallelizing some calls. But most of the calls are query calls. We only do the update when we find the subaccount with the empty balance.

Reply:
sure, I'll add a comment